open-metadata-resources/open-metadata-deployment/charts/lab/README.md

```diff
@@ -138,16 +137,36 @@ by deleting the deployment and running the installation again. This will wipe ou
 metadata across the lab Egeria servers, remove all messages from the Kafka bus used in the cohort,
 reset the Jupyter notebooks to their original clean state, etc.
 
-To delete the deployment, simply run:
+To delete the deployment, simply run this for Helm3:
 
 ```shell script
-$ helm delete labs --purge
+$ helm delete lab
+```
+Or if using Helm2:
+```$xslt
```

Comment:
This should be `shell script` as well, not `$xslt` (?)

Reply:
Agree. done. thanks